cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -1,5 +1,7 @@
 package relayer
 
+// DONTCOVER
```

Comment:
Whats the reasoning with all the `DONTCOVER` ?

Reply:
The code coverage will now notice this, and excluded the file from code coverage stats.

Reply:
Sorry my question was more why are we excluding these files?

Reply:
These are core files in ebrelayer.  Can't mock these without significant work that is better focused elsewhere.  There are a variety of other files excluded from code coverage (via settings we copied from cosmos-sdk).  Like all top-level main.go files, as well as client directories that do boiler-plate io plumbing.  That is essentially what we are doing here.

Reply:
oh ok I get it now.  Thanks